tokio/src/util/trace.rs

```diff
@@ -5,7 +5,10 @@ cfg_trace! {
     use pin_project_lite::pin_project;
 
     use tracing::Span;
-    // A future, stream, sink, or executor that has been instrumented with a `tracing` span.
+    // A future, stream, sink, or executor that has been instrumented with a
+    // `tracing` span.
+
```

Comment:
Is it intentional that this isn't a doc comment? And I think the extra newline here needs to be removed for the doc comment to actually be associated with the struct. Or maybe even put inside of `pin_project!`.

Reply:
The comment was also not even entirely _accurate_, since this version only works for `Future`s --- that's just a copy-paste error. My bad!